### PR TITLE
Fix $FlowFixMes from TabBarIOS

### PR DESCRIPTION
**Pull request by cdlewis**
## Motivation

There are a number of $FlowFixMe statements in TarBarIOS.ios.js as a result of recent Flow upgrades introducing new errors/warnings. I had a stab at removing these statements and introducing what are hopefully sensible types.

## Test Plan

Only types were changed so `yarn flow` should be sufficient.

## Release Notes

[INTERNAL] [MINOR] [TarBarIOS] - |Fix $FlowFixMes|

### CODE REVIEW

File: Libraries/Components/TabBarIOS/TabBarIOS.ios.js [hunk 1]
```diff
@@ -20,19 +20,22 @@ const ViewPropTypes = require('ViewPropTypes');
...
+class TabBarIOS extends React.Component<ViewProps & {
+  style?: StyleObj,
+  unselectedTintColor?: string,
+  tintColor?: string,
+  unselectedItemTintColor?: string,
+  barTintColor?: string,
   barStyle?: 'default' | 'black',
   translucent?: boolean,
   itemPositioning?: 'fill' | 'center' | 'auto',
+  children: React.Element<any> | React.Element<any>[],
```
Comment (TheSavior) [inline Libraries/Components/TabBarIOS/TabBarIOS.ios.js]: This should probably be [`React.Node`](https://flow.org/en/docs/react/children/)

**Comment by cdlewis** [comment]
@TheSavior I updated this PR to use React.Node as suggested.

**Comment by TheSavior** [comment]
Thank you very much for contributing this. We love PRs improving our flow types!

**Comment by cdlewis** [comment]
Thanks @TheSavior <3

**Pull request closed without merging**